README.md

```diff
@@ -38,8 +38,8 @@ Oryx is a framework for building composable storefronts, providing a rich librar
 
 ## Installation
 
-TODO
+Please refer to [Set up Oryx guide](https://docs.spryker.com/docs/scos/dev/front-end-development/{{page.version}}/oryx/getting-started/set-up-oryx.html).
```

Comment:
It's not correct as long as this guide is written for the boilerplate, not for the mono-repo.

Reply:
I'll probably take it over and write some simple instructions.